modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/CacheChangesProcessor.java

```diff
@@ -126,10 +127,10 @@ private void sendCacheInfo() {
     private Collection<CacheSqlMetadata> getCacheSqlMetadata() {
         GridCacheProcessor cacheProc = ctx.cache();
 
-        List<CacheSqlMetadata> cachesMetadata = new ArrayList<>();
+        Collection<CacheSqlMetadata> cachesMetadata = new HashSet<>();
 
         for (Map.Entry<String, DynamicCacheDescriptor> item : cacheProc.cacheDescriptors().entrySet()) {
-            if (item.getValue().sql()) {
+            if (!item.getValue().schema().entities().isEmpty() && !isSystemCache(item.getKey())) {
```

Comment:
I think that we can add "sql" flag to CacheSqlMetadata. This flag can be used on UI to group SQL table or add some visual sign that table was created via SQL, for example.

Reply:
Add createBySql parameter.